### DISCUSSION TITLE
Delete Developers guide in Readme of the packages

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#398] [Ideas]
In the Readme.md files of most packages, there is still the developer's guide from the template. For example, here: [autora-theorist-bsr](https://github.com/AutoResearch/autora-theorist-bsr). It doesn't make much sense to keep this in there (if at all, we should make a contribution guide so that people can contribute to these packages).

**Comment by musslick** [comment]
I agree that we should remove the developer's guide from the template!

Also: Moving this to "Ideas" since it's more of a suggestion than a question.
